solhint.js

```diff
@@ -17,6 +17,11 @@ function init() {
         .description('Linter for Solidity programming language')
         .action(execMainAction);
 
+    program
+        .usage('[options] <file> [...other_files]')
+        .option('-q, --quiet', 'report errors only - default: false')
```

Comment:
This is duplicating the previous block. I think you just need to add the `.option` line in that block.

Reply:
Perfect, I will modify it.